spec/shared_behaviors/taggable.rb

```diff
@@ -139,7 +149,12 @@ def tag_link(s)
         '#-initialhyphen'        => '-initialhyphen',
         '#-initialhyphen tag'    => '-initialhyphen',
         '#-initial-hyphen'       => '-initial-hyphen',
-        "\u202a#\u200eUSA\u202c" => 'USA'
+        "\u202a#\u200eUSA\u202c" => "USA",
+        "test #ինչո՞ւ test"      => "ինչո՞ւ",
+        "test #այո՜ո test"       => "այո՜ո",
+        "test #սեւ֊սպիտակ test"  => "սեւ֊սպիտակ",
+        "test #գժանո՛ց test"     => "գժանո՛ց",
+        "test #മലയാണ്മ test"      => "മലയാണ്മ"
```

Comment:
Hmm, indeed, anyway, the `=` is in column 34 and this one is in column 35, I'd say make them the same column regardless.